{"uuid":"3dd9b1d4-72a1-4e4c-b534-2089bec1cc32","historyId":"bc5221230495cf20a767e3de59d65f3a","fullName":"Tests.CreateMerchantScreenTest.TC_PSA_0083","labels":[{"name":"package","value":"Tests.CreateMerchantScreenTest"},{"name":"testClass","value":"Tests.CreateMerchantScreenTest"},{"name":"testMethod","value":"TC_PSA_0083"},{"name":"parentSuite","value":"Default Suite"},{"name":"suite","value":"SAPortalTest_Selenium_POM"},{"name":"subSuite","value":"Tests.CreateMerchantScreenTest"},{"name":"host","value":"DESKTOP-MCJ49E3"},{"name":"thread","value":"13672@DESKTOP-MCJ49E3.main(1)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"}],"links":[],"name":"Validation of adding an existing Email used before for another merchant","status":"failed","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"The following asserts failed:\n\tNO Error message appears despite creating a new merchant with a previously used Email expected [true] but found [false]","trace":"java.lang.AssertionError: The following asserts failed:\n\tNO Error message appears despite creating a new merchant with a previously used Email expected [true] but found [false]\r\n\tat org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:43)\r\n\tat Tests.CreateMerchantScreenTest.TC_PSA_0083(CreateMerchantScreenTest.java:648)\r\n\tat java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)\r\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:577)\r\n\tat org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)\r\n\tat org.testng.internal.Invoker.invokeMethod(Invoker.java:583)\r\n\tat org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)\r\n\tat org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)\r\n\tat org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)\r\n\tat org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)\r\n\tat org.testng.TestRunner.privateRun(TestRunner.java:648)\r\n\tat org.testng.TestRunner.run(TestRunner.java:505)\r\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:455)\r\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)\r\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)\r\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:364)\r\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)\r\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)\r\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)\r\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1137)\r\n\tat org.testng.TestNG.runSuites(TestNG.java:1049)\r\n\tat org.testng.TestNG.run(TestNG.java:1017)\r\n\tat com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)\r\n\tat com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:109)\r\n"},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1667733170462,"stop":1667733184908}